src/main/java/taxi/dao/DriverDaoImpl.java

```diff
@@ -17,13 +17,15 @@
 public class DriverDaoImpl implements DriverDao {
     @Override
     public Driver create(Driver driver) {
-        String query = "INSERT INTO drivers (name, license_number) "
-                + "VALUES (?, ?)";
+        String query = "INSERT INTO drivers (name, license_number,login,password) "
+                + "VALUES (?, ?,?,?)";
```

Comment:
Please do not forget about formatting, add spaces after comas.
Check similar cases in other classes.
```suggestion
        String query = "INSERT INTO drivers (name, license_number, login, password) "
                + "VALUES (?, ?, ?, ?)";
```